src/erlang_ls_completion_provider.erl

```diff
@@ -91,18 +94,22 @@ find_completion( Prefix
       variables(Document);
     %% Check for "[...] atom"
     [{atom, _, _} | _] ->
-      modules(Prefix) ++ functions(Document, false);
+      modules() ++ functions(Document, false);
```

Comment:
`++ keywords()` should be included here as well, right?

Reply:
Yes.